### PR TITLE
Polish translations for task C

### PR DESCRIPTION
**Pull request by MilyMilo**
Everything should be fine by now.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@MilyMilo, it looks like you've already signed the Dropbox CLA. Thanks!

File: translations/Polish/special-terms.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* Group PM - Czat grupowy (Konwersacja would be too formal)
+* Realm - Domena (Królestwo wouldn't fit the context)
+* Stream - Kanał (Strumień wouldn't fit the context)
+* Topic - Wątek 
+* Private/Invite-Only Stream - Prywatny kanał (Kanał na zaproszenie seems too long)
+* Public Stream - Publiczny kanał
+* Bot - Bot (Robot just doesn't sound right in my opinion)
+* Integration  - Integracja
+* Notification - Powiadomienie
+* Alert Word - Słowo alarm
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: I think this one is too literal - alert words are words you get notifications for, just like you'd for an @-mention. Something more like "oznaczone/otagowane słowo" would probably convey the meaning better.

File: translations/Polish/special-terms.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+* View - Widok
+* Home - Strona głowna
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: główna

File: translations/Polish/special-terms.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+* Emoji - Emotka (singular) or Emotki (plural)
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: `Emotka` or `emotikonka` is different than `emoji` - I think it's `emoji`.

File: translations/Polish/special-terms.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+# Phrases
+
+* Subscribe to a stream/Unsubscribe from a stream - Zasubskrybuj kanał
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: Missing second translation.

File: translations/Polish/special-terms.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+* Narrow to - Zwęzić do
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: Zawęzić

File: translations/Polish/special-terms.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* Filter - Filtr
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: This can be both a noun and a verb.

File: translations/Polish/special-terms.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+* Mute/Unmute - Wycisz / Anuluj wyciszenie
+* Deactivate/Reactivate - Dezaktywuj/Reaktywuj
+* Search - Wyszukaj
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: This can be both a noun and a verb.

File: translations/Polish/special-terms.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+* Pin - Odznaka / Kołek (Depending on the context)
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: Pin means `przypiąć` as a verb and it's more like `wpinka/pinezka/przypinka` in Polish as a noun.

File: translations/Polish/special-terms.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+* Mention/@mention - Wspomnij/@oznacz
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: Mention is a verb here, I think `wspomnieć` or `oznaczyć` would work. `@mention` is more of a noun.

File: translations/Polish/special-terms.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
+# Terms
+English word | Polish word
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: You didn't provide example usage or explanation for most of the terms, could you look into that?

As per task description:
> Translate every term in this list to language in the format English word: *language word*, [explanation for the decision], [example usage in the Zulip context].

Also, I don't think the format you use is `English word | Polish word`.

File: translations/Polish/special-terms.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+* I want - Chcę 
+* User - Użytkownik 
+* Person/People - Osoba (singular) / Ludzie (plural)
+
+# Other
+
+* You - ty
+* We - my
+
+
```
Comment (trueskawka) [inline translations/Polish/special-terms.md]: Superfluous whitespace.

**Comment by trueskawka** [comment]
Hi @MilyMilo!

Thanks for working on this!

I've added some comments, could you address those?

Your PR name doesn't match the task description, could you fix it?

Could you also squash your commits? This PR should have only one commit.

Thanks!

**Comment by Lyla-Fischer** [comment]
It's been almost a year. I don't think that @MilyMilo is coming back to this. (although she should feel free to open up another pull request if she wants to get her work saved as a 2016 submission)

**Pull request closed without merging**